app/controllers/concerns/tabs.rb

```diff
@@ -0,0 +1,18 @@
+module Concerns::Tabs
+  extend ActiveSupport::Concern
```

Comment:
I think this isn't needed as you're not relying on any other module as a dependency, so it can be a standard module - @Levia what do you think?